hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/DataSourceOptions.scala

```diff
@@ -191,6 +191,16 @@ object DataSourceWriteOptions {
     .withAlternatives("hoodie.datasource.write.storage.type")
     .withDocumentation("The table type for the underlying data, for this write. This can’t change between writes.")
 
+   val HIVE_SYNC_USE_KERBEROS: ConfigProperty[Boolean] = ConfigProperty
+    .key("hoodie.datasource.hive_sync.use_kerberos")
+    .defaultValue(false)
+    .withDocumentation("Whether to use Kerberos authentication.")
+
+  val HIVE_SYNC_KERBEROS_PRINCIPAL: ConfigProperty[String] = ConfigProperty
+    .key("hive_sync.kerberos_principal")
+    .noDefaultValue()
```

Comment:
hoodie.datasource.hive_sync.kerberos_principal ? And move this option together with the other hive sync options.

Reply:
Has been adjusted

Reply:
The node running the Flink task that starts Async Hive is the corresponding client. We need to do three things:
1. Add parameter 'hive_sync.use_kerberos' = 'true' to HUDI SQL
2.Add parameter Kerberos_principal '=' Principal of the hive you are connecting to '
3. Each client has been authenticated by Kerberos
A simple example is as follows：


> > 1. Perform Kerberos authentication on the client
>  Each server connected to Hive is a Hive client.

> How can we do that ? What do you mean by the client ?
The node running the Flink task that starts Async Hive is the corresponding client. We need to do three things:
1. Add parameter 'hive_sync.use_kerberos' = 'true' to HUDI SQL
2.Kerberos_principal '=' Principal of the hive you are connecting to '
3. Each client has been authenticated by Kerberos
A simple example

CREATE TABLE t2(
Uuid VARCHAR (20),
···
)
with(
'connector' = 'hudi',
'path' = 'hdfs://ip:8020/warehouse/hudi/t2',
'hive_sync.enable'='true',
'hive_sync.metastore.uris'='thrift://ip:9083',
'hive_sync.use_kerberos' = 'true',
'hive_sync.kerberos_principal' = 'hive/_HOST@BIGDATA'
)